packages/js/customer-effort-score/src/utils/customer-effort-score-exit-page.ts

```diff
@@ -9,7 +9,13 @@ import { getQuery } from '@woocommerce/navigation';
 /**
  * Internal dependencies
  */
-import { ALLOW_TRACKING_OPTION_NAME } from './constants';
+import { ALLOW_TRACKING_OPTION_NAME } from '../constants';
+
+interface AdminWindow extends Window {
+	pagenow?: string;
+	adminpage?: string;
+}
+declare let window: AdminWindow;
```

Comment:
Needed to add this to take care of TS error about `pagenow` and `adminpage` not being defined.